src/domains/stac.ts

```diff
@@ -42,10 +42,10 @@ const downloadAssets = (concept: Collection | Granule) => {
   return (concept.relatedUrls ?? [])
     .filter((relatedUrl) => relatedUrl["type"] === RelatedUrlType.GET_DATA)
     .reduce((downloadAssets, relatedUrl, idx, available) => {
-      const [entry, display] = available.length > 1 ? [`_${idx}`, ` [${idx}]`] : ["", ""];
-
+      const display = available.length > 1 ? ` [${idx}]` : "";
```

Comment:
ultra nitpick but

```suggestion
      const display = available.length > 1 ? `[${idx}]` : "";
```

Reply:
Actually this space was needed to add a space between `Direct Download` and `[1]` in `Direct Download [1]`. It'll look a bit squished.

Reply:
Okay its fine that way. If its for that then I would think you could just add it to `title: `Direct Download ${display}`,`?